wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -673,6 +673,11 @@ def locales():
     ])
 
 
+@register.simple_tag()
+def slim_sidebar_enabled():
```

Comment:
Maybe you could reuse this function in the if statement for the other two tags too (just a quick amendment to avoid repeating logic, `if slim_sidebar_enabled()` reads okay to me)?

Reply:
Good idea, thanks!